doc/CCPPtechnical/source/CodeManagement.rst

```diff
@@ -32,23 +32,22 @@ https://github.com/NCAR/FMS
 
 Users have read-only access to these repositories and as such cannot accidentally destroy any important (shared) branches of these authoritative repositories.
 
-Some of these repositories are public (no GitHub account required) and some are private. The public repositories (ccpp-framework, ccpp-physics, and Flexible Modeling System - FMS) may be used directly to read or create forks. Write permission is generally restricted, however. The private repositories require access - please send a request and your GitHub username to gmtb-help@ucar.edu.
-
-The primary development by GMTB, including the latest CCPP developments, are maintained in the following branches:
+Some of these repositories are public (no GitHub account required) and some are private. The public repositories (ccpp-framework, ccpp-physics, NEMS, and Flexible Modeling System - FMS) may be used directly to read or create forks. Write permission is generally restricted, however. The private repositories require access - please send a request and your GitHub username to gmtb-help@ucar.edu.
 
+The following branches are recommended for CCPP users and developers:
 
 +---------------------------------------------+----------------------+
 | Repository (GMTB development version)       |    Branch name       |
 +=============================================+======================+
-| https://github.com/NCAR/NEMSfv3gfs          |    gmtb/ccpp         |
+| https://github.com/NCAR/NEMSfv3gfs          |   master             |
 +---------------------------------------------+----------------------+
-| https://github.com/NCAR/FV3                 |    gmtb/ccpp         |
+| https://github.com/NCAR/FV3                 |   master             |
 +---------------------------------------------+----------------------+
 | https://github.com/NCAR/ccpp-physics        |   master             |
 +---------------------------------------------+----------------------+
 | https://github.com/NCAR/ccpp-framework      |   master             |
 +---------------------------------------------+----------------------+
-| https://github.com/NCAR/NEMS                |   gmtb/ccpp          |
+| https://github.com/NCAR/NEMS                |   develop            |
 +---------------------------------------------+----------------------+
 | https://github.com/NCAR/FMS                 |    GFS-FMS           |
 +---------------------------------------------+----------------------+
```

Comment:
Maybe unusual, but equivalent. We are always keeping the branches listed above in sync, i.e. they head of each of them is guaranteed to work with all the others.

Reply:
Maybe we should list branches and tags. The branch information is needed if someone is going to create a pull request for a certain branch of NEMSfv3gfs and its submodules.

Reply:
But tags will change much more often than the technical documentation!

Reply:
This documentation is for top of master. Target audience is model developers, which will likely be modifying one or more submodules of NEMSfv3gfs.  Because they will be creating PR of innovations, they cannot work from fixed tags.
All repositories under NEMSfv3gfs are submodules. Those are ccpp-physics, ccpp-framework, FV3, NEMS etc. Developers will be modifying one or more of these submodules.
Note that we also offer a  public release of CCPP with the Single-Column Model. In that case, we point to tags because the public release is stable and does not change.